src/Controls/tests/TestCases.Shared.Tests/Tests/FlakyTestAttribute.cs

```diff
@@ -1,13 +1,84 @@
 ﻿using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+using NUnit.Framework.Internal.Commands;
 
-namespace Microsoft.Maui.TestCases.Tests;
-
-public class FlakyTestAttribute : IgnoreAttribute
+namespace Microsoft.Maui.TestCases.Tests
 {
-	public FlakyTestAttribute() : base(nameof(FlakyTestAttribute))
-	{
-	}
-	public FlakyTestAttribute(string reason) : base(reason)
+	// Custom NUnit attribute to mark a test as flaky, allowing retries (by default 2).
```

Comment:
Even though its internal can we make this an API doc comment? So with 3 slashes `///` so that it shows up when you use this attribute?